azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -170,12 +176,17 @@ private HadoopSecurityManager_H_2_0(final Props props)
     this.conf = new Configuration();
     this.conf.setClassLoader(ucl);
 
-    if (props.containsKey(FS_HDFS_IMPL_DISABLE_CACHE)) {
-      logger.info("Setting " + FS_HDFS_IMPL_DISABLE_CACHE + " to "
-          + props.get(FS_HDFS_IMPL_DISABLE_CACHE));
-      this.conf.setBoolean(FS_HDFS_IMPL_DISABLE_CACHE,
-          Boolean.valueOf(props.get(FS_HDFS_IMPL_DISABLE_CACHE)));
-    }
+    // Disable FileSystem Cache for HadoopSecurityManager
+    this.conf.setBoolean(FS_HDFS_IMPL_DISABLE_CACHE, true);
+    this.conf.setBoolean(FS_FAILOVER_IMPL_DISABLE_CACHE, true);
+    // Get the default scheme
+    final String defaultFS = conf.get(CommonConfigurationKeysPublic.FS_DEFAULT_NAME_KEY);
+    final String scheme = new Path(defaultFS).toUri().getScheme();
+    // Construct the property name
+    final String FS_DEFAULT_IMPL_DISABLE_CACHE =
+            "fs." + scheme + IMPL_DISABLE_CACHE_SUFFIX;
```

Comment:
You need a null check here. Also, does using a static Configuration make sense here?